content/integrations/azure.md

```diff
@@ -60,6 +60,8 @@ After logging in, locate the settings page on the left side of the screen.
 The text in the red box shown in sceenshot above is your tenent name. Please only include text between parentheses.
 
 #### Unable to login
+Make sure that you have created an Active Directory, you can use its default domain, for more information see the [azure documentation][5].
```

Comment:
the wording seems off here. Lets hold off on merging until after Tristan finishes testing the new workflow